08-confidence-intervals.Rmd

```diff
@@ -950,9 +947,9 @@ $$\overline{x} \pm 1.96 \cdot SE = (\overline{x} - 1.96 \cdot SE, \, \overline{x
 
 Computation of the 95% confidence interval can once again be done by piping the `bootstrap_distribution` data frame we created into the `get_confidence_interval()` function. However, this time we set the first `type` argument to be `"se"`. Second, we must specify the `point_estimate` argument in order to set the center of the confidence interval. We set this to be the sample mean of the original sample of 50 pennies of `r x_bar_point <- x_bar %>% pull(mean_year) %>% round(2); x_bar_point`.
 
-<!-- point_estimate = 1995.44 -->
-
 ```{r}
+x_bar <- pennies_sample %>% 
+  summarize(mean_year = mean(year))
```

Comment:
Does it make sense to redefine `x_bar` here of to just say that `x_bar` was calculated earlier in the chapter and remind them of the code? It doesn't really matter in the long run, but wanna make sure we try to maintain consistency on things like this.

Reply:
good point. will fix